pkg/common/providers/mock/mock.go

```
@@ -234,3 +234,8 @@ func (m *MockProvider) SetVersionList(list *spi.VersionList) {
 func (m *MockProvider) Type() string {
 	return "mock"
 }
+
+// ExtendExpiry extends the expiration time of an existing cluster
+func (m *MockProvider) ExtendExpiry(clusterID string, hours uint64, minutes uint64, seconds uint64) error {
+	return nil
```

Let's return an unsupported error here too.